### PR TITLE
Garlic Message Bus recipes

### PR DESCRIPTION
**Pull request by imaximius**
Q | A
-- | --
License | MIT

### CODE REVIEW

File: garlic/bus/1.0/config/packages/garlic.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+enqueue:
```
Comment (Pierstoval) [inline garlic/bus/1.0/config/packages/garlic.yaml]: This corresponds to another package than Garlic, you shouldn't post this config here, as there is another one for Enqueue.

Comment (imaximius) [reply garlic/bus/1.0/config/packages/garlic.yaml]: Thanx! It just mistake. I'll change it

Comment (imaximius) [reply garlic/bus/1.0/config/packages/garlic.yaml]: @Pierstoval Already changed!

**Comment by imaximius** [comment]
Bundle has the command that up and running consumers and producers to the message queue. I thought it'll be comfortable already have supervisor config for this purposes

**Comment by Nyholm** [comment]
I do understand why you need supervisor. Im just cant figure out why you want that config in `my-symfony-project/config/`. Should that config be in `/etc/supervisor/config` or something like that?

File: garlic/bus/1.0/manifest.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "bundles": {
+        "Garlic\\Bus\\GarlicBusBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "RABBIT_HOST": "rabbit",
```
Comment (Nyholm) [inline garlic/bus/1.0/manifest.json]: Where are these environment variables used?

**Pull request closed without merging**